api/pkg/apis/projectcalico/v3/tier.go

```diff
@@ -48,6 +48,10 @@ type TierSpec struct {
 	// last.  Tiers with identical order will be applied in alphanumerical order based
 	// on the Tier "Name".
 	Order *float64 `json:"order,omitempty" protobuf:"bytes,1,opt,name=order"`
+	// EndOfTierAction specifies the action applied to workloads selected by a policy in the tier,
+	// but not rule matched the workload's traffic.
+	// [Default: Deny]
+	EndOfTierAction Action `json:"endOfTierAction,omitempty" validate:"omitempty,oneof=Deny Pass"`
```

Comment:
I thin `DefaultAction` is a better name for this field. @caseydavenport WDYT?

Reply:
Yeah, I like DefaultAction :+1: